.pre-commit-config.yaml

```diff
@@ -43,13 +43,5 @@ repos:
         types: [markdown]
         files: ^tasks/.*\.md$
 
-      # redundant? already validated by validate-task-frontmatter?
-      # should combine them
-      - id: validate-tasks
-        name: Validate task metadata
-        description: Validates YAML frontmatter in task files
-        entry: ./scripts/tasks.py check
-        language: system
-        types: [markdown]
-        pass_filenames: false
-        #files: ^tasks/.*\.md$
```

Comment:
This should be included, or silently pass/skip if tasks.py doesn't exist/broken symlink.